sql/core/src/test/resources/sql-tests/inputs/predicate-functions.sql

```diff
@@ -39,3 +43,4 @@ select 2.0 <= '2.2';
 select 0.5 <= '1.5';
 select to_date('2009-07-30 04:17:52') <= to_date('2009-07-30 04:17:52');
 select to_date('2009-07-30 04:17:52') <= '2009-07-30 04:17:52';
+select to_date('2017-03-01') <= to_timestamp('2017-03-01 00:00:01');
```

Comment:
A little hard to track the exact changes. Let us move all the new tests to the end.

Reply:
+1 it is really confusing to look at the diff